src/Modal.jsx

```diff
@@ -31,6 +36,28 @@ class Modal extends React.Component {
 		}
 	}
 
+	/**
+	 * @param {String} scrollPosition window scroll position
+	 * @returns {String} CSS value for setting modal margin-top
+	 */
+	getModalPosition(scrollPosition) {
+		const viewportHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
+
+		if (scrollPosition < viewportHeight) {
+			return DEFAULT_MARGIN_TOP;
```

Comment:
This won't be the only component that will need to be aware of media queries, so there's a PR open to add utils for this kind of thing: https://github.com/meetup/meetup-web-components/pull/149